src/components/Footer/Footer.js

```diff
@@ -1,7 +1,147 @@
-import React from 'react';
+import React, { useState } from 'react';
+import FooterDataModal from '../Modal/FooterDataModal';
+import {
+  StyleFooter,
+  FooterContainer,
+  FooterInner,
+  FooterWrapper,
+  Ul,
+  Li,
+  FooterDiv,
+  SlideDiv,
+  MapBtn,
+  MapBtnSpan,
+  MapBtnLink,
+  FooterWrapperDiv,
+  FooterLink,
+} from './StyleFooter';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faGlobe,
+  faChevronUp,
+  faMap,
+  faListUl,
+} from '@fortawesome/free-solid-svg-icons';
 
 const Footer = () => {
-  return <div>Footer</div>;
+  const [isMap, setIsMap] = useState(false);
+  const [isOpenModal, setIsOpenModal] = useState(false);
+
+  const handleModal = () => {
+    setIsOpenModal(!isMap);
+  };
+  return (
+    <StyleFooter>
+      <MapBtn onClick={() => setIsMap(!isMap)}>
+        {isMap === true ? (
```

Comment:
```suggestion
        {isMap ? (
```
isMap 자체가 boolean 값이기 때문에 위와 같이 하셔도 됩니다!